pod/src/list/list_trait.rs

```diff
@@ -27,6 +27,12 @@ pub trait List {
     /// Returns a read-only slice of the items currently in the list.
     fn as_slice(&self) -> &[Self::Item];
 
+    /// Returns a reference to an element at a given index or `None` if the
+    /// index is out of bounds.
+    fn get(&self, index: usize) -> Option<&Self::Item> {
+        self.as_slice().get(index)
+    }
```

Comment:
nit: I feel the same about this -- if you already have `as_slice()`, what does this add?